worlds/hk/docs/setup_en.md

```diff
@@ -1,15 +1,32 @@
 # Hollow Knight for Archipelago Setup Guide
 
 ## Required Software
-* Download and unzip the Scarab Mod Manager from the [Scarab GitHub Releases page](https://github.com/fifty-six/Scarab/releases).
-* A legal copy of Hollow Knight, not purchased or played through XBox Game Pass.
-    * Unfortunately, the Game Pass version is not currently compatible with mods.
+* Download and unzip the Scarab+ Mod Manager from the [Scarab+ website](https://themulhima.github.io/Scarab/).
+* A legal copy of Hollow Knight.
+
+## Optional Software
+* Archipelago Map Mod from Scarab+
 
 ## Installing the Archipelago Mod using Scarab
-1. Launch Scarab and ensure it locates your Hollow Knight installation directory.
-2. Click the "Install" checkbox near the "Archipelago" mod entry.
+1. Launch Scarab+ and ensure it locates your Hollow Knight installation directory.
+2. Click the "Install" button near the "Archipelago" mod entry.
 3. Launch the game, you're all set!
 
+### Using Scarab+ with XBox Game Pass
+1. Enter the XBox app and move your mouse over "Hollow Knight" on the left sidebar. 
+2. Click the three points then click "Manage".
+3. Go to the "Files" tab and select "Browse...". 
+4. Click "Hollow Knight", then "Content", then click the path bar and copy it.
+5. Run Scarab+ as an administrator and, when it asks you for the path, paste what you copied in step 4.
```

Comment:
Scarab+ will actually detect it automatically if you install in the default location of `C:\XboxGames` (or any X:\XboxGames for any drive letter), so if that's the case the user wouldn't need to follow the steps in this section

It might be worth labelling this as something like "If Scarab+ doesn't find it automatically